pkg/webhook/admission/pod/storage_initializer_injector_test.go

```diff
@@ -30,6 +31,22 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	StorageInitializerDefaultCPURequest    = "100m"
+	StorageInitializerDefaultCPULimit      = "1"
+	StorageInitializerDefaultMemoryRequest = "200Mi"
+	StorageInitializerDefaultMemoryLimit   = "1Gi"
+)
+
+var (
+	storageInitializerConfig = &StorageInitializerConfig{
+		CpuRequest:    StorageInitializerDefaultCPURequest,
```

Comment:
Why not just define all of these inline?

Reply:
Could you please clarify what do you mean by define those inline?

Reply:
By inline, I meant do something like `resource.MustParse(config.CONFIGKEY)` which should clean up some of this boilerplate. I think this will slightly change your tests, but will overall make it cleaner. If you try it and disagree, feel free to leave as is.

The other thing I just noticed is that we should be using these values from the configmap, rather than defining the defaults here.

Reply:
@ellis-bigelow I defined those default values here so they can be used in this unit test. The when we inject the default resource limit in the storage container I used the values from configmap.

Reply:
Define them inline doesn't seem to make it simpler. But I do get your point that you don't want the same resource spec copy paste to all the test cases. So I abstract them out to make the tests cleaner.